WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -114,69 +117,27 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
 {
     [WordPressAppDelegate fixKeychainAccess];
 
+    // Basic networking setup
+    [self setupReachability];
+    
+    // Set the main window up
+    CGRect bounds = [[UIScreen mainScreen] bounds];
```

Comment:
I think some of these lines might not be necessary for now as as a result of https://github.com/wordpress-mobile/WordPress-iOS/pull/3937, but we can keep them for now and just patch them out later. No need to introduce anything else new with this.